e2e_deployment_files/stress_deployment.template.windows.json

```diff
@@ -111,14 +111,122 @@
                 "value": "<LoadGen.MessageFrequency>"
               },
               "transportType": {
-                "value": "<LoadGen4.TransportType>"
+                "value": "<TransportType4>"
               }
             },
             "settings": {
               "image": "<Container_Registry>/microsoft/azureiotedge-load-gen:<Build.BuildNumber>-windows-<Architecture>",
               "createOptions": ""
             }
           },
+          "twinTester1": {
```

Comment:
Please confirm with Varun, if twin test is required to run in stress test.

Reply:
Varun instructed me to only add to stress if it could handle once a second. I ran for an hour at this rate (also with larger updates) and there were no failures, so we should keep in stress.